docs/source/using/troubleshooting.adoc

```diff
@@ -51,21 +51,39 @@ $ systemctl enable virtlogd
 ----
 
 [[domain-minishift-already-exists]]
-=== Error starting the VM: ... operation failed: domain 'minishift' already exists ...
+=== KVM: `Error starting the VM: ...operation failed: domain 'minishift' already exists...`
```

Comment:
This title is too long. We should just add the essential part of the error message, e.g. 

KVM:  'minishift' already exists...